server/index.js

```diff
@@ -47,7 +47,7 @@ app.get('/', (req, res) => {
 app.get('/news/:article', (req, res) => {
   res.send(index);
 });
-app.get('/report/:reportCode([A-Za-z0-9]+)/:fightId([0-9]+)?:fightName(-[^/]+)?/:playerName([^/]{2,})?/:tab([A-Za-z0-9-]+)?', (req, res) => {
+app.get('/report/:reportCode([A-Za-z0-9]+)/:fightId([0-9]+)?:fightName(-[^/]+)?/:playerId([0-9]+)?:playerName(-[^/]{2,})?/:tab([A-Za-z0-9-]+)?', (req, res) => {
```

Comment:
Did you check the playername regex? I think this regex will incorrectly include the dash in the player name param.

Reply:
I see my mistake here, updated the regex so the dash isn't matched with the player name.